pkg/kubelet/eviction/helpers.go

```diff
@@ -204,7 +204,7 @@ func parseThresholdStatement(signal evictionapi.Signal, val string) (*evictionap
 		if percentage < 0 {
```

Comment:
My vote is that calling this variable `percentage` is still slightly confusing... to me, `percentage` indicates a value from 0 to 100. I wonder if `ratio` would be a more appropriate term here?

Reply:
According to my understanding, firstly, we need to keep the decimal value of `func parsePercentage` due to the `struct ThresholdValue` definition. So we restore the modification of  `fuc parsePercentage`. Otherwise, the return value should between 0 to 1 by defination. Therefore we modify the calling in `func parseThresholdStatement` from percentage > 100 to percentage > 1.

At the same time, I think that you means the variable `percentage` itself is confusing, we need use `ratio `or other suitable term to replace it. Do I understand it correctly?

Reply:
Yup! I believe I agree with you around needing to keep the decimal value. However, I find `percentage` to be a confusing variable name given that its value must be between 0 and 1, instead of between 0 and 100.

What about `percentageAsDecimal` as a variable name?

Reply:
It seems that `percentageAsDecimal` can express the meaning clearly enough. And I have updated this commit with the new variable name already. 

The other func calling of `parsePercentage ` is its FT `TestParsePercentage`. Do we need to modify the return variable `value `at the same time?

Reply:
In my mind, it also makes sense to update the function name to be `parsePercentageAsDecimal`.

/retest

Reply:
`parsePercentageAsDecimal` sounds good, someone can know its function from its name.

And after considering it carefully, I think `value` in FT TestParsePercentage do not need to modify. So I will just modify the func name as `parsePercentageAsDecimal`. What‘s your opinion?

Reply:
Yup, I think just changing the function name makes sense.

Reply:
In addition, if the function name will be changed to `parsePercentageAsDecimal`, the return value  in `func parseThresholdStatement` may use simple term like decimalValue. 

What is your point of view? Should we modify both the function name and the variable which received the return value to *`PercentageAsDecimal`, or just for the function name?

Reply:
If its just used in the test, I think that `value` should be fine. Tbh, I think either is fine :)

Reply:
I have updated the func name and the calling already. Do we need to do something else?